great-idea/css/index.css

```diff
@@ -62,3 +62,114 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+/*My code starts here*/
+
+/*Universal*/
+
+html{
+	font-size: 16px;
+	font-family: Helvetica, sans-serif;
+}
+
+body{
+	display: flex;
+	justify-content: center;
+}
+
+.page-container{
+	width: 80%;
+}
+
+/*Header*/
+
+header{
+	position: fixed;
```

Comment:
I like that you ran with an idea, might improve upon it with some transparency to the background. Interestingly if you for example used `background: #FFF3;` or a variation, you might beable to get creative with the white clipping off when things scroll underneath of it.